src/distdl/nn/pad.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
```

Comment:
If we are using `F.pad()`, can we get rid of this?

When padding works through the halo we will deprecate `PadNd` and `UnpadNd`.

Reply:
OK, so now I think I understand...This is only handling padding on the left and right boundary workers, the necessary padding to make the layer match torch.  In this case, I think this logic should be explicitly in a layer that uses it, not a general pad layer.